cli.js

```diff
@@ -33,9 +33,11 @@ const findConfig = (configFromCli) => {
 
   return new Promise((resolve, reject) => {
     if (configFromCli) {
+      // console.log('cFC')
```

Comment:
TODO: Remove these commented lines.

Reply:
I'll do that in the next PR as these were to see what broke after bumping `commander` and `cosmiconfig`.